app/javascript/utilities/color/contrastValidator.js

```diff
@@ -0,0 +1,32 @@
+import { WCAGColorContrast } from './WCAGColorContrast';
```

Comment:
The reason that I create a `contrastValidator` utility and a `WCAGColorContrast` one is so that if we decide to change the underlying `WCAGColorContrast` code that we use to determine the constrast ratio, then we only need to replace the library and reference the new one in this file. 

Our stimulus controller doesn't know or depend on the library or underlying implementation we're using to determine the contrast ratio.